Makefile

```
@@ -0,0 +1,17 @@
+default: simulate
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)